.github/workflows/windows-build.yml

```
@@ -151,7 +136,21 @@ jobs:
             tar -xvpf ${{ env.DEV_DRIVE_WORKSPACE }}\cache.tar
             rm ${{ env.DEV_DRIVE_WORKSPACE }}\cache.tar
```

```suggestion
            rm ${{ env.DEV_DRIVE_WORKSPACE }}\cache.tar
            ${{ O3DE_COMPILER_CACHE_PATH }} --zero-stats
```

If you move "Setup ccache" back to its original spot, this will just run the stat zeroing command after the archive is restored